packages/trpc/src/context-app-router.ts

```diff
@@ -2,6 +2,10 @@ import { SALEOR_API_URL_HEADER, SALEOR_AUTHORIZATION_BEARER_HEADER } from "@sale
 import { inferAsyncReturnType } from "@trpc/server";
 import { FetchCreateContextFnOptions } from "@trpc/server/adapters/fetch";
 
+/**
```

Comment:
I think we can remove this file - only Stripe app was using this context anyway

Reply:
done